chain/db/validator.go

```diff
@@ -88,3 +109,58 @@ func (b *BandDB) ClearOldVotes(currentHeight int64) error {
 	}
 	return nil
 }
+
+func (b *BandDB) GetValidator(validator sdk.ValAddress) (Validator, bool) {
+	validatorStruct := Validator{OperatorAddress: validator.String()}
+	err := b.tx.First(&validatorStruct).Error
+	return validatorStruct, err == nil
+}
+
+func (b *BandDB) handleMsgEditValidator(msg staking.MsgEditValidator) error {
+	validator, isFound := b.GetValidator(msg.ValidatorAddress)
+	if !isFound {
+		return fmt.Errorf(fmt.Sprintf("validator %s has not exist.", msg.ValidatorAddress.String()))
+	}
+
+	if msg.Description.Moniker != "[do-not-modify]" {
```

Comment:
Why do we type out string here? Let's use `DoNotModifyDesc` variable?

Reply:
https://github.com/cosmos/cosmos-sdk/blob/release/v0.37.8/x/staking/alias.go